tests/debugger/test_debugger_base.py

```diff
@@ -2,32 +2,63 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-from utils import scenarios, interfaces, weblog, features, missing_feature, context
+from utils import interfaces
 from utils.tools import logger
-
-
-def validate_probes(expected_probes):
-    def get_probes_map():
-        agent_logs_endpoint_requests = list(interfaces.agent.get_data(path_filters="/api/v2/logs"))
-        probe_hash = {}
-
-        for request in agent_logs_endpoint_requests:
-            content = request["request"]["content"]
-            if content is not None:
-                for content in content:
+from packaging import version
+import json
+import re
+
+_CONFIG_PATH = "/v0.7/config"
+_DEBUGER_PATH = "/api/v2/debugger"
+_LOGS_PATH = "/api/v2/logs"
+_TRACES_PATH = "/api/v0.2/traces"
+
+
+def read_data():
```

Comment:
suggestion: `read_data()` -> `read_diagnostic_data`

Reply:
You're right, much better. Done here 00db5f86e9cf9d1c0801cbd927724ad53e1d10d5